lib/chef/chef_fs/file_system.rb

```diff
@@ -68,7 +68,7 @@ def list_from(entry, &block)
                 list_from(exact_child, &block)
               end
 
-            # Otherwise, go through all children and find any matches
+              # Otherwise, go through all children and find any matches
```

Comment:
its consistent, it keeps everything inside the block at the same indentation level.